### PR TITLE
Fixed non-UTF8 characters.

### PR DESCRIPTION
**Pull request by fjpanag**
## Summary

Fixed several non-UTF8 characters throughout the source code.

## Impact

N/A

## Testing

N/A

### CODE REVIEW

**Review by acassis — APPROVED** [review]

**Comment by acassis** [comment]
@fjpanag nice! How did you search for non-UTF8 chars?

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by fjpanag** [comment]
> @fjpanag nice! How did you search for non-UTF8 chars?

I used `cvt2utf` to do the conversion.

Is it possible to include this in CI, so non-UTF8 characters are no more introduced in the codebase?

**Comment by xiaoxiang781216** [comment]
> > @fjpanag nice! How did you search for non-UTF8 chars?
> 
> I used `cvt2utf` to do the conversion.
> 
> Is it possible to include this in CI, so non-UTF8 characters are no more introduced in the codebase?

it's good ideal to add some guard in CI. BTW, should we apply the same change to apps/ folder too.

**Comment by xiaoxiang781216** [comment]
@fjpanag let's fix the follow style issue?
```
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/src/a1x/hardware/a10_memorymap.h:289:1: warning: #include outside of 'Included Files' section
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/boards/arm/lpc31xx/ea3[13](https://github.com/apache/incubator-nuttx/actions/runs/3135514943/jobs/5091280431#step:3:14)1/src/lpc31_mem.c:106:78: error: Long line found
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/boards/arm/lpc31xx/ea3152/src/lpc31_mem.c:106:78: error: Long line found
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/boards/arm/sama5/sama5d3-xplained/src/sam_ethernet.c:169:79: error: Long line found
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/boards/arm/sama5/sama5d3-xplained/src/sam_sdram.c:[14](https://github.com/apache/incubator-nuttx/actions/runs/3135514943/jobs/5091280431#step:3:15)0:78: error: Long line found
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/boards/arm/sama5/sama5d3x-ek/src/sam_ethernet.c:[16](https://github.com/apache/incubator-nuttx/actions/runs/3135514943/jobs/5091280431#step:3:17)9:79: error: Long line found
```

**Comment by fjpanag** [comment]
@xiaoxiang781216 I will take a look on the style errors.  
I will run the tool on `apps/` too.

Can you help with the CI script?

File: arch/arm/src/armv7-a/arm.h [hunk 1]
```diff
@@ -19,10 +19,10 @@
  ****************************************************************************/
 
 /* References:
- *  "Cortex-A5� MPCore, Technical Reference Manual", Revision: r0p1,
- *   Copyright � 2010 ARM. All rights reserved. ARM DDI 0434B (ID101810)
- *  "ARM� Architecture Reference Manual, ARMv7-A and ARMv7-R edition",
- *   Copyright � 1996-1998, 2000, 2004-2012 ARM. All rights reserved.
+ *  "Cortex-A5™ MPCore, Technical Reference Manual", Revision: r0p1,
```
Comment (xiaoxiang781216) [inline arch/arm/src/armv7-a/arm.h:22]: is this conversion right?

**Comment by fjpanag** [comment]
BTW the warning `Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/src/a1x/hardware/a10_memorymap.h:289:1: warning: #include outside of 'Included Files' section` is a false alarm.

File: arch/arm/src/armv7-a/arm.h (hunk 1 above)
Comment (fjpanag) [reply arch/arm/src/armv7-a/arm.h:22]: I think so? What do you thing is the problem?

Comment (xiaoxiang781216) [reply arch/arm/src/armv7-a/arm.h:22]: it looks good on my smart phone, but wrong on my PC. The locale config may have some problem in my PC.

**Pull request merged**